### ISSUE TITLE
Build with --androidTypings got nothing

### ISSUE THREAD

**Issue by hosikiti** [NativeScript/nativescript-cli#3381]
### Did you verify this is a real problem by searching the [NativeScript Forum]
yes. But I'm not sure this is an environment depenedent issue or NativeScript's.

### Tell us about the problem
Even if I build my project with `tns build android  --androidTypings`, no *.d.ts file is generated on project root. My log says 'Project successfully built.' and indeed, apk file is generated successfully.

I suspect that something is wrong with `asbg` because I got following message with `tns build android --androidTypings --log debug`

```
> Configure project :asbg
false
```

But I don't know what caused this 'false'  and how to investigate it.

### Which platform(s) does your issue occur on?
Android

### Please provide the following version numbers that your issue occurs with:

```
┌──────────────────┬─────────────────┬────────────────┬──────────────────┐
│ Component        │ Current version │ Latest version │ Information      │
│ nativescript     │ 3.4.0           │ 3.4.2          │ Update available │
│ tns-core-modules │ 3.4.0           │ 3.4.1          │ Update available │
│ tns-android      │ 3.4.1           │ 3.4.1          │ Up to date       │
│ tns-ios          │ 3.4.1           │ 3.4.1          │ Up to date       │
└──────────────────┴─────────────────┴────────────────┴──────────────────┘
```

### Please tell us how to recreate the issue in as much detail as possible.

Sample-groceries project with following dependencies in package.json to recreate the issue.

```
  "dependencies": {
    "@angular/common": "~5.2.0",
    "@angular/compiler": "~5.2.0",
    "@angular/core": "~5.2.0",
    "@angular/forms": "~5.2.0",
    "@angular/http": "~5.2.0",
    "@angular/platform-browser": "~5.2.0",
    "@angular/platform-browser-dynamic": "~5.2.0",
    "@angular/router": "~5.2.0",
    "email-validator": "^1.0.7",
    "nativescript-angular": "~5.2.0",
    "nativescript-disable-bitcode": "^1.0.0",
    "nativescript-google-analytics": "^0.4.1",
    "nativescript-gradient": "^2.0.1",
    "nativescript-grid-view": "^3.2.2",
    "nativescript-image-cache": "^1.1.4",
    "nativescript-image-filters": "^2.0.0",
    "nativescript-imagepicker": "^4.0.1",
    "nativescript-iqkeyboardmanager": "~1.2.0",
    "nativescript-localize": "^2.0.1",
    "nativescript-localstorage": "^1.1.5",
    "nativescript-pro-ui": "^3.3.0",
    "nativescript-pulltorefresh": "^2.0.2",
    "nativescript-social-share": "~1.4.0",
    "nativescript-theme-core": "~1.0.2",
    "nativescript-unit-test-runner": "^0.3.3",
    "nativescript-webview-interface": "^1.4.2",
    "reflect-metadata": "~0.1.8",
    "rxjs": "^5.5.0",
    "tns-core-modules": "3.4.0",
    "zone.js": "^0.8.4"
  },
  "devDependencies": {
    "@angular/compiler-cli": "~5.2.0",
    "@ngtools/webpack": "~1.9.1",
    "babel-traverse": "6.24.1",
    "babel-types": "6.24.1",
    "babylon": "6.17.0",
    "codelyzer": "^3.0.1",
    "copy-webpack-plugin": "~4.3.0",
    "css-loader": "~0.28.7",
    "extract-text-webpack-plugin": "~3.0.2",
    "filewalker": "^0.1.3",
    "jasmine-core": "^2.6.2",
    "karma": "^1.7.0",
    "karma-jasmine": "^1.0.2",
    "karma-nativescript-launcher": "^0.4.0",
    "lazy": "1.0.11",
    "tslint": "^5.4.2",
    "webpack": "~3.10.0",
    "webpack-bundle-analyzer": "^2.9.1",
    "webpack-sources": "~1.1.0",
    "nativescript-css-loader": "~0.26.0",
    "nativescript-custom-entitlements": "^0.2.0",
    "nativescript-dev-typescript": "^0.6.0",
    "nativescript-dev-webpack": "^0.9.0",
    "nativescript-worker-loader": "~0.8.1",
    "node-sass": "^4.5.3",
    "node-sass-chokidar": "0.0.3",
    "raw-loader": "~0.5.1",
    "resolve-url-loader": "~2.1.0",
    "s3-cli": "^0.13.0",
    "sass-loader": "~6.0.6",
    "tns-platform-declarations": "^3.1.1",
    "typescript": "~2.4.2",
  }
```

**Comment by Pip3r4o** [comment]
@hosikiti I can confirm that no android typings are currently generated when passing the `--androidTypings` flag to the build command. I'll investigate and let you know what I find out.

The `false` log is unrelated to the androidTypings flag and functionality, and will be removed in the following android runtime to avoid causing confusion.

**Comment by Pip3r4o** [comment]
@hosikiti we found the cause and have addressed the problem. The fix will be available with the upcoming 4.0 release of the android-runtime package.

**Comment by hosikiti** [comment]
@Pip3r4o That is great. Thank you. Looking forward to 4.0.
